### ISSUE TITLE
Mainnet node is stuck at block 15085353

### ISSUE THREAD

**Issue by 1scrooge** [hyperledger/besu#4070]
I running besu node in docker. 
```
version: "3.5"
services:
  ethereum:
    image: hyperledger/besu:22.7
    restart: always
    network_mode: host
    volumes:
      - ./data:/opt/besu/database/
    command:
      --network=mainnet
      --rpc-http-enabled
      --sync-mode=fast
      --host-allowlist=*
      --nat-method=upnp
      --Xdns-enabled=true
      --rpc-http-api=ETH,NET,CLIQUE,DEBUG,MINER,NET,PERM,ADMIN,EEA,TXPOOL,PRIV,WEB3
      --logging=DEBUG
```
Tonight the node is stuck at block 15085353. Besu version was 22.4.3, today I've updated to 22.7 but it doesn't help
Some log excerpts:

```
ethereum_1  | 2022-07-07 08:45:26.894+00:00 | nioEventLoopGroup-3-6 | DEBUG | EthProtocolManager | Peer 0x200778efd1f8cc2360... PeerReputation 100, validated? false, disconnected? false has mismatched network id: 190260
ethereum_1  | 2022-07-07 08:45:26.894+00:00 | nioEventLoopGroup-3-6 | DEBUG | EthProtocolManager | Disconnect - Outbound - 0x10 SUBPROTOCOL_TRIGGERED - PeerInfo{version=5, clientId='Geth/senmarkchain/v1.10.0-stable(quorum-v22.1.0)/linux-amd64/go1.18.2', capabilities=[eth/64, eth/65, eth/66, istanbul/64, istanbul/99, istanbul/100, snap/1], port=0, nodeId=0x200778efd1f8cc236004414ead5f028a12868a7007a7cf8fdc276f074d4d9caa6260a826766e666cc1ffe0f8cdb30f4a365f066cb0cd39eb72f9a4264e660d16} - 9 peers left
ethereum_1  | 2022-07-07 08:45:26.894+00:00 | nioEventLoopGroup-3-6 | DEBUG | SnapProtocolManager | Disconnect - Outbound - 0x10 SUBPROTOCOL_TRIGGERED - PeerInfo{version=5, clientId='Geth/senmarkchain/v1.10.0-stable(quorum-v22.1.0)/linux-amd64/go1.18.2', capabilities=[eth/64, eth/65, eth/66, istanbul/64, istanbul/99, istanbul/100, snap/1], port=0, nodeId=0x200778efd1f8cc236004414ead5f028a12868a7007a7cf8fdc276f074d4d9caa6260a826766e666cc1ffe0f8cdb30f4a365f066cb0cd39eb72f9a4264e660d16} - 9 peers left
ethereum_1  | 2022-07-07 08:45:26.924+00:00 | nioEventLoopGroup-3-1 | DEBUG | DeFramer | Invalid incoming message
ethereum_1  | io.netty.handler.codec.DecoderException: org.hyperledger.besu.ethereum.rlp.RLPException: Expected current item to be a list, but it is: BYTE_ELEMENT (at bytes 0-1: [04])
ethereum_1  | 	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:480)
ethereum_1  | 	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:279)
ethereum_1  | 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
ethereum_1  | 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
ethereum_1  | 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
ethereum_1  | 	at org.hyperledger.besu.ethereum.p2p.rlpx.connections.netty.AbstractHandshakeHandler.channelRead0(AbstractHandshakeHandler.java:135)
ethereum_1  | 	at org.hyperledger.besu.ethereum.p2p.rlpx.connections.netty.AbstractHandshakeHandler.channelRead0(AbstractHandshakeHandler.java:44)
ethereum_1  | 	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:99)
ethereum_1  | 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
ethereum_1  | 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
ethereum_1  | 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
ethereum_1  | 	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
ethereum_1  | 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
ethereum_1  | 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
ethereum_1  | 	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
ethereum_1  | 	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)
ethereum_1  | 	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:722)
ethereum_1  | 	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658)
ethereum_1  | 	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584)
ethereum_1  | 	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496)
ethereum_1  | 	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
ethereum_1  | 	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
ethereum_1  | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
ethereum_1  | 	at java.base/java.lang.Thread.run(Thread.java:829)
ethereum_1  | Caused by: org.hyperledger.besu.ethereum.rlp.RLPException: Expected current item to be a list, but it is: BYTE_ELEMENT (at bytes 0-1: [04])
ethereum_1  | 	at org.hyperledger.besu.ethereum.rlp.AbstractRLPInput.error(AbstractRLPInput.java:202)
ethereum_1  | 	at org.hyperledger.besu.ethereum.rlp.AbstractRLPInput.enterList(AbstractRLPInput.java:448)
ethereum_1  | 	at org.hyperledger.besu.ethereum.rlp.BytesValueRLPInput.enterList(BytesValueRLPInput.java:23)
ethereum_1  | 	at org.hyperledger.besu.ethereum.rlp.AbstractRLPInput.enterList(AbstractRLPInput.java:432)
ethereum_1  | 	at org.hyperledger.besu.ethereum.rlp.BytesValueRLPInput.enterList(BytesValueRLPInput.java:23)
ethereum_1  | 	at org.hyperledger.besu.ethereum.p2p.rlpx.wire.messages.DisconnectMessage$Data.readFrom(DisconnectMessage.java:83)
ethereum_1  | 	at org.hyperledger.besu.ethereum.p2p.rlpx.wire.messages.DisconnectMessage.getReason(DisconnectMessage.java:61)
ethereum_1  | 	at org.hyperledger.besu.ethereum.p2p.rlpx.connections.netty.DeFramer.decode(DeFramer.java:180)
ethereum_1  | 	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:510)
ethereum_1  | 	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:449)
ethereum_1  | 	... 23 more
ethereum_1  | 2022-07-07 08:45:26.924+00:00 | nioEventLoopGroup-3-1 | DEBUG | RlpxAgent | Failed to connect to peer 0x0391f1fab08e52000b1bdee23d6530993dfa147f6997a6c751aff9a6dc4811badc8cb869c4e55898199820db8e03158e3f96e507eb759d8364c16437b37eff8b: io.netty.handler.codec.DecoderException: org.hyperledger.besu.ethereum.rlp.RLPException: Expected current item to be a list, but it is: BYTE_ELEMENT (at bytes 0-1: [04])
ethereum_1  | 2022-07-07 08:45:27.028+00:00 | nioEventLoopGroup-3-8 | DEBUG | EthProtocolManager | Peer 0xcace77feebe682a5a2... PeerReputation 100, validated? false, disconnected? false has mismatched network id: 100
ethereum_1  | 2022-07-07 08:45:27.028+00:00 | nioEventLoopGroup-3-8 | DEBUG | EthProtocolManager | Disconnect - Outbound - 0x10 SUBPROTOCOL_TRIGGERED - PeerInfo{version=5, clientId='Nethermind/v1.12.7-0-3b419f196-20220620/X64-Linux/6.0.3', capabilities=[eth/62, eth/63, eth/64, eth/65, eth/66], port=30305, nodeId=0xcace77feebe682a5a24fbb204da4e3d60c10281cea5ea7997dad261ba992e4d4971239fe39ce104689fb49addaff9a294b0ba903668a528c109083ebab04f37c} - 8 peers left
ethereum_1  | 2022-07-07 08:45:27.115+00:00 | nioEventLoopGroup-3-8 | DEBUG | EthProtocolManager | Peer 0xd42ac8091924cb979b... PeerReputation 100, validated? false, disconnected? false has mismatched network id: 5
ethereum_1  | 2022-07-07 08:45:27.115+00:00 | nioEventLoopGroup-3-8 | DEBUG | EthProtocolManager | Disconnect - Outbound - 0x10 SUBPROTOCOL_TRIGGERED - PeerInfo{version=5, clientId='Geth/v1.7.4-stable-d56ca12f/linux-amd64/go1.11.2', capabilities=[eth/63, eth/62], port=0, nodeId=0xd42ac8091924cb979ba490d679ed77568aec9524f3269ece1dee128c538ea277f6c48321f5f7aba7f1ee28489c4f865994aaa735f38901a1cf79f237d02ae45f} - 7 peers left
ethereum_1  | 2022-07-07 08:45:27.205+00:00 | nioEventLoopGroup-3-9 | DEBUG | RlpxAgent | Outbound connection established to peer: 0x156a72000740f0e59692a17811d32d1d513ecc8655c40c3b89b03efe622e2d87fdd2ccc011aa60690836ada508197d2b9b1827055f0f2e4fd02091480d75a45b
ethereum_1  | 2022-07-07 08:45:27.205+00:00 | nioEventLoopGroup-3-9 | DEBUG | EthProtocolManager | Sending status message to Peer 0x156a72000740f0e596... PeerReputation 100, validated? false, disconnected? false.
ethereum_1  | 2022-07-07 08:45:27.499+00:00 | nioEventLoopGroup-3-9 | DEBUG | ApiHandler | Received Wire PING
ethereum_1  | 2022-07-07 08:45:27.500+00:00 | nioEventLoopGroup-3-9 | DEBUG | EthProtocolManager | Peer 0x156a72000740f0e596... PeerReputation 100, validated? false, disconnected? false has mismatched network id: 111
ethereum_1  | 2022-07-07 08:45:27.500+00:00 | nioEventLoopGroup-3-9 | DEBUG | EthProtocolManager | Disconnect - Outbound - 0x10 SUBPROTOCOL_TRIGGERED - PeerInfo{version=5, clientId='OpenEthereum/v3.2.2-rc.1-stable-582bca3-20210331/x86_64-linux-gnu/rustc1.51.0', capabilities=[eth/63, eth/64, par/1, par/2], port=30300, nodeId=0x156a72000740f0e59692a17811d32d1d513ecc8655c40c3b89b03efe622e2d87fdd2ccc011aa60690836ada508197d2b9b1827055f0f2e4fd02091480d75a45b} - 7 peers left
ethereum_1  | 2022-07-07 08:45:27.640+00:00 | nioEventLoopGroup-3-3 | DEBUG | HandshakeHandlerOutbound | Wrote initial crypto handshake message to /119.45.194.112:30322.
ethereum_1  | 2022-07-07 08:45:27.742+00:00 | vert.x-eventloop-thread-1 | DEBUG | RecursivePeerRefreshState | Bonding round timed out
ethereum_1  | 2022-07-07 08:45:27.742+00:00 | vert.x-eventloop-thread-1 | DEBUG | RecursivePeerRefreshState | Initiating neighbours round with 3 candidates from 834 tracked nodes
ethereum_1  | 2022-07-07 08:45:27.849+00:00 | vert.x-eventloop-thread-1 | DEBUG | RecursivePeerRefreshState | Received neighbours packet with 12 neighbours
ethereum_1  | 2022-07-07 08:45:27.897+00:00 | nioEventLoopGroup-3-3 | DEBUG | DeFramer | Disconnecting because no capabilities are shared: PeerInfo{version=5, clientId='Gvns/v3.2.0-unstable/linux-amd64/go1.12.4', capabilities=[vns/63, vns/64], port=0, nodeId=0x4407fcd613e853bbe46737529383a343c439ecadd0e28ddf95b6094a1b8572118781a59521066b707ab7ba7f7d5c1d11ed11a05db4e3f916969e3b85070f5a78}
ethereum_1  | 2022-07-07 08:45:27.897+00:00 | nioEventLoopGroup-3-3 | DEBUG | RlpxAgent | Failed to connect to peer 0x4407fcd613e853bbe46737529383a343c439ecadd0e28ddf95b6094a1b8572118781a59521066b707ab7ba7f7d5c1d11ed11a05db4e3f916969e3b85070f5a78: org.hyperledger.besu.ethereum.p2p.network.exceptions.IncompatiblePeerException: No shared capabilities
ethereum_1  | 2022-07-07 08:45:27.904+00:00 | vert.x-eventloop-thread-1 | DEBUG | RecursivePeerRefreshState | Received neighbours packet with 12 neighbours
ethereum_1  | 2022-07-07 08:45:27.982+00:00 | vert.x-eventloop-thread-1 | DEBUG | RecursivePeerRefreshState | Received neighbours packet with 12 neighbours
ethereum_1  | 2022-07-07 08:45:27.983+00:00 | vert.x-eventloop-thread-1 | DEBUG | RecursivePeerRefreshState | Initiating bonding round with 16 candidates
ethereum_1  | 2022-07-07 08:45:27.985+00:00 | vert.x-worker-thread-2 | DEBUG | PongPacketData | read PONG enr from scalar
ethereum_1  | 2022-07-07 08:45:27.985+00:00 | vert.x-worker-thread-2 | DEBUG | PingPacketData | read PING enr as long scalar
ethereum_1  | 2022-07-07 08:45:27.990+00:00 | nioEventLoopGroup-3-8 | DEBUG | HandshakeHandlerOutbound | Wrote initial crypto handshake message to /49.12.129.62:20540.
ethereum_1  | 2022-07-07 08:45:27.993+00:00 | nioEventLoopGroup-3-8 | DEBUG | DeFramer | Peer enode://6d03754587991862b0a5c95dff825fbd4d9466c3bbd3cbc11cd8b7e5822f552f96326b31b75af634ce6d9b13f1b11be972687f584ea9a91bb8f623e674d4a5b4@49.12.129.62:20540 disconnected before sending HELLO.  Reason: 0x04 TOO_MANY_PEERS
ethereum_1  | 2022-07-07 08:45:27.993+00:00 | nioEventLoopGroup-3-8 | DEBUG | RlpxAgent | Failed to connect to peer 0x6d03754587991862b0a5c95dff825fbd4d9466c3bbd3cbc11cd8b7e5822f552f96326b31b75af634ce6d9b13f1b11be972687f584ea9a91bb8f623e674d4a5b4: org.hyperledger.besu.ethereum.p2p.network.exceptions.PeerDisconnectedException: Peer disconnected for reason: 0x04 TOO_MANY_PEERS
ethereum_1  | 2022-07-07 08:45:28.011+00:00 | vert.x-worker-thread-0 | DEBUG | PongPacketData | read PONG enr from scalar
ethereum_1  | 2022-07-07 08:45:28.011+00:00 | vert.x-worker-thread-0 | DEBUG | PingPacketData | read PING enr as long scalar
ethereum_1  | 2022-07-07 08:45:28.017+00:00 | nioEventLoopGroup-3-10 | DEBUG | HandshakeHandlerOutbound | Wrote initial crypto handshake message to /194.147.58.198:39797.
ethereum_1  | 2022-07-07 08:45:28.036+00:00 | nioEventLoopGroup-3-10 | DEBUG | DeFramer | Disconnecting because no capabilities are shared: PeerInfo{version=5, clientId='energi3/v3.1.3-stable/linux-amd64/go1.17.9', capabilities=[eth/70], port=0, nodeId=0x05cee1437554d79804564ab47af673f2d75c5ebf29cf2035c2848fb696ebf40268ec464085c0405fe02ddf79b98f4cef927b6b84a51655d106a6a540abd0f1fe}
ethereum_1  | 2022-07-07 08:45:28.037+00:00 | nioEventLoopGroup-3-10 | DEBUG | RlpxAgent | Failed to connect to peer 0x05cee1437554d79804564ab47af673f2d75c5ebf29cf2035c2848fb696ebf40268ec464085c0405fe02ddf79b98f4cef927b6b84a51655d106a6a540abd0f1fe: org.hyperledger.besu.ethereum.p2p.network.exceptions.IncompatiblePeerException: No shared capabilities
ethereum_1  | 2022-07-07 08:45:28.040+00:00 | nioEventLoopGroup-3-1 | DEBUG | HandshakeHandlerOutbound | Wrote initial crypto handshake message to /95.217.12.250:30303.
ethereum_1  | 2022-07-07 08:45:28.055+00:00 | vert.x-worker-thread-4 | DEBUG | PongPacketData | read PONG enr from scalar
ethereum_1  | 2022-07-07 08:45:28.055+00:00 | vert.x-worker-thread-4 | DEBUG | PingPacketData | read PING enr as long scalar
ethereum_1  | 2022-07-07 08:45:28.070+00:00 | nioEventLoopGroup-3-1 | DEBUG | RlpxAgent | Outbound connection established to peer: 0xc30ae4f766f799a8d8fb1e87f0c32c57469bf03f61d49e6550ad7cce63877d7e9e49807871e034cb786e4614bf33b21c5d75440878b72ac346ed896a17126baf
ethereum_1  | 2022-07-07 08:45:28.070+00:00 | nioEventLoopGroup-3-1 | DEBUG | EthProtocolManager | Sending status message to Peer 0xc30ae4f766f799a8d8... PeerReputation 100, validated? false, disconnected? false.
ethereum_1  | 2022-07-07 08:45:28.074+00:00 | vert.x-worker-thread-8 | DEBUG | PongPacketData | read PONG enr from scalar
ethereum_1  | 2022-07-07 08:45:28.075+00:00 | vert.x-worker-thread-3 | DEBUG | PingPacketData | read PING enr as long scalar
ethereum_1  | 2022-07-07 08:45:28.077+00:00 | vert.x-worker-thread-17 | DEBUG | PongPacketData | read PONG enr from scalar
ethereum_1  | 2022-07-07 08:45:28.078+00:00 | vert.x-worker-thread-17 | DEBUG | PingPacketData | read PING enr as long scalar
ethereum_1  | 2022-07-07 08:45:28.082+00:00 | vert.x-worker-thread-1 | DEBUG | PingPacketData | read PING enr as long scalar
ethereum_1  | 2022-07-07 08:45:28.083+00:00 | vert.x-worker-thread-1 | DEBUG | PongPacketData | read PONG enr from scalar
ethereum_1  | 2022-07-07 08:45:28.088+00:00 | vert.x-worker-thread-2 | DEBUG | PingPacketData | read PING enr as long scalar
ethereum_1  | 2022-07-07 08:45:28.089+00:00 | vert.x-worker-thread-2 | DEBUG | PongPacketData | read PONG enr from scalar
ethereum_1  | 2022-07-07 08:45:28.089+00:00 | vert.x-worker-thread-2 | DEBUG | PingPacketData | read PING enr as long scalar
ethereum_1  | 2022-07-07 08:45:28.089+00:00 | vert.x-worker-thread-9 | DEBUG | PongPacketData | read PONG enr from scalar
ethereum_1  | 2022-07-07 08:45:28.096+00:00 | nioEventLoopGroup-3-1 | DEBUG | EthProtocolManager | Peer 0xc30ae4f766f799a8d8... PeerReputation 100, validated? false, disconnected? false has mismatched network id: 9372
```
any ideas?

**Comment by daniellehrner** [comment]
Could you provide your complete log file? The part that you have shared does not indicate why the node is stuck.

**Comment by 1scrooge** [comment]
@daniellehrner 
[besu.zip](https://github.com/hyperledger/besu/files/9063089/besu.zip)
Full log after enabled DEBUG

**Comment by daniellehrner** [comment]
The only thing that I can see in the log files is that the node is trying to find peers, but does not succeed. Did you have other versions of Besu running in the past and could connect to peers?

**Comment by 1scrooge** [comment]
@daniellehrner No, I'm running besu node for the 1st time. It syncing from scratch and stuck at block 5085353

**Comment by 1scrooge** [comment]
@daniellehrner Hi. I launched besu from scratch. It stuck again, but other block 15121585. Full log
[besu15121585.log](https://github.com/hyperledger/besu/files/9109769/besu15121585.log)

**Comment by garyschulte** [comment]
that log makes it look like besu is not actually stuck but is downloading the worldstate.  See the log line about 1 minute before the end of the file:
```
{"log":"2022-07-13 07:48:10.237+00:00 | EthScheduler-Services-7 (requestCompleteTask) | INFO  | CompleteTaskStep | Downloaded 590616834 world state nodes. At least 16751 nodes remaining.\n","stream":"stdout","time":"2022-07-13T07:48:10.237206725Z"}
```

The other "found common ancestor" log lines are just noisy peering log lines.  Did this node eventually finish downloading the world state and/or select a newer pivot block than 15121585?

**Comment by 1scrooge** [comment]
@garyschulte No. still 15121585

**Comment by matkt** [comment]
could you share us what is your current log now ?

**Comment by matkt** [comment]
I found that you have a stacktrace related to Upnp, can be related to your peering issue 

```
{“log”:“2022-07-07 08:09:54.399+00:00 | main | DEBUG | NatService | Caught exception while trying to query NAT external IP address (ignoring)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.40438944Z”}
{“log”:“java.util.concurrent.TimeoutException\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404431372Z”}
{“log”:“\u0009at java.base/java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1886)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404438983Z”}
{“log”:“\u0009at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2021)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404445225Z”}
{“log”:“\u0009at org.hyperledger.besu.nat.NatService.queryExternalIPAddress(NatService.java:145)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404451058Z”}
{“log”:“\u0009at org.hyperledger.besu.ethereum.p2p.discovery.PeerDiscoveryAgent.start(PeerDiscoveryAgent.java:154)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404456842Z”}
{“log”:“\u0009at org.hyperledger.besu.ethereum.p2p.network.DefaultP2PNetwork.start(DefaultP2PNetwork.java:235)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404462355Z”}
{“log”:“\u0009at org.hyperledger.besu.ethereum.p2p.network.NetworkRunner.start(NetworkRunner.java:80)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404467998Z”}
{“log”:“\u0009at org.hyperledger.besu.Runner.startEthereumMainLoop(Runner.java:139)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404473479Z”}
{“log”:“\u0009at org.hyperledger.besu.cli.BesuCommand.run(BesuCommand.java:1431)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404494417Z”}
{“log”:“\u0009at picocli.CommandLine.executeUserObject(CommandLine.java:1939)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404500864Z”}
{“log”:“\u0009at picocli.CommandLine.access$1300(CommandLine.java:145)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404505985Z”}
{“log”:“\u0009at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404510951Z”}
{“log”:“\u0009at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404516064Z”}
{“log”:“\u0009at picocli.CommandLine$RunLast.handle(CommandLine.java:2314)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404521056Z”}
{“log”:“\u0009at picocli.CommandLine$AbstractParseResultHandler.handleParseResult(CommandLine.java:2172)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404526247Z”}
{“log”:“\u0009at picocli.CommandLine.parseWithHandlers(CommandLine.java:2559)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404531348Z”}
{“log”:“\u0009at org.hyperledger.besu.cli.util.ConfigOptionSearchAndRunHandler.handle(ConfigOptionSearchAndRunHandler.java:54)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.40453648Z”}
{“log”:“\u0009at org.hyperledger.besu.cli.util.ConfigOptionSearchAndRunHandler.handle(ConfigOptionSearchAndRunHandler.java:31)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404541781Z”}
{“log”:“\u0009at picocli.CommandLine$AbstractParseResultHandler.handleParseResult(CommandLine.java:2172)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404547197Z”}
{“log”:“\u0009at picocli.CommandLine.parseWithHandlers(CommandLine.java:2559)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404552436Z”}
{“log”:“\u0009at org.hyperledger.besu.cli.BesuCommand.parse(BesuCommand.java:1586)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404557588Z”}
{“log”:“\u0009at org.hyperledger.besu.cli.BesuCommand.parse(BesuCommand.java:1391)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404562829Z”}
{“log”:“\u0009at org.hyperledger.besu.Besu.main(Besu.java:49)\n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.404568097Z”}
{“log”:“2022-07-07 08:09:54.399+00:00 | main | WARN  | NatService | Unable to query NAT external IP address. Using the fallback value : 127.0.0.1 \n”,“stream”:“stdout”,“time”:“2022-07-07T08:09:54.40457556Z”}
```

**Comment by garyschulte** [comment]
fixed with #4175

**Comment by githubswisschain** [comment]
@garyschulte I started besu 22.7.06, but it got stuck again at block 15267497
last log
```
ethereum_1  | 2022-08-13 07:38:53.392+00:00 | nioEventLoopGroup-3-6 | WARN  | FullSyncTargetManager | Disconnecting PeerId 0xbde9875e4bcd758647d64dae4458f3c2ad437377f4e1846bfc746b20ffed088ca4bcd6472bd5f89e64c0e30b5e44c46678c57c42a79f96a27b660eefbce6f69d, reputation PeerReputation 170, validated? true, disconnected? false, client: Geth/v1.10.19-stable/linux-amd64/go1.17.10, connection 39476003, enode enode://bde9875e4bcd758647d64dae4458f3c2ad437377f4e1846bfc746b20ffed088ca4bcd6472bd5f89e64c0e30b5e44c46678c57c42a79f96a27b660eefbce6f69d@18.138.242.161:30303 because world state is not available at common ancestor at block 15267497 (0x6e7b0900cd580f81b65454eba1e0301d43f70c674f8cee772c60c8d4c0540427)
ethereum_1  | 2022-08-13 07:55:30.850+00:00 | nioEventLoopGroup-3-10 | WARN  | FullSyncTargetManager | Disconnecting PeerId 0x1d20cd90bb4269d693a9a9beff79f5b5398f67e89aa4964ab6a85f16cdc1a971b09626e7c9e5cc2c2d510d4725f3e2d8e32d19452238f5af3b90e649d06a6b07, reputation PeerReputation 115, validated? true, disconnected? false, client: Geth/v1.10.21-stable-67109427/linux-amd64/go1.18.4, connection 1050401768, enode enode://1d20cd90bb4269d693a9a9beff79f5b5398f67e89aa4964ab6a85f16cdc1a971b09626e7c9e5cc2c2d510d4725f3e2d8e32d19452238f5af3b90e649d06a6b07@180.183.236.56:30303 because world state is not available at common ancestor at block 15267497 (0x6e7b0900cd580f81b65454eba1e0301d43f70c674f8cee772c60c8d4c0540427)
ethereum_1  | 2022-08-13 07:58:35.773+00:00 | nioEventLoopGroup-3-10 | WARN  | FullSyncTargetManager | Disconnecting PeerId 0xf5d5463c5eacf750d254619ba1c47790cd0987b7995abd84658695677e3f1d0ed00e70687d3e420e4e6d32ce2a95900cf0f2adf0182cd3f242e813bbed1ca9f2, reputation PeerReputation 170, validated? true, disconnected? false, client: Geth/v1.10.19-stable/linux-amd64/go1.17.10, connection 1085860792, enode enode://f5d5463c5eacf750d254619ba1c47790cd0987b7995abd84658695677e3f1d0ed00e70687d3e420e4e6d32ce2a95900cf0f2adf0182cd3f242e813bbed1ca9f2@54.90.34.206:30303 because world state is not available at common ancestor at block 15267497 (0x6e7b0900cd580f81b65454eba1e0301d43f70c674f8cee772c60c8d4c0540427)
ethereum_1  | 2022-08-13 08:08:10.568+00:00 | nioEventLoopGroup-3-4 | WARN  | FullSyncTargetManager | Disconnecting PeerId 0x9fc3d924e437b1009f22d95cf906fc966feba3b8cb5d2d1637dab7cf237169b777094047807d966f31d20924165f9498ff6b2446183169dcaf362f3b78b3d81d, reputation PeerReputation 133, validated? true, disconnected? false, client: Geth/v1.10.19-stable-23bee162/linux-amd64/go1.18.1, connection 1885997855, enode enode://9fc3d924e437b1009f22d95cf906fc966feba3b8cb5d2d1637dab7cf237169b777094047807d966f31d20924165f9498ff6b2446183169dcaf362f3b78b3d81d@207.199.157.98:20010 because world state is not available at common ancestor at block 15267497 (0x6e7b0900cd580f81b65454eba1e0301d43f70c674f8cee772c60c8d4c0540427)
ethereum_1  | 2022-08-13 08:15:02.153+00:00 | nioEventLoopGroup-3-1 | WARN  | FullSyncTargetManager | Disconnecting PeerId 0xf9e2fa7344f4238a9c985224585459d8ab710a23d0dd2176c3743470dda1f3fbd8a83d5e194285f367c7d0c83c3a97c4c4529f94c29cdb6e674283ba75ce2f03, reputation PeerReputation 168, validated? true, disconnected? false, client: Geth/v1.10.19-stable-175b1496/linux-amd64/go1.18.2, connection 1532737282, enode enode://f9e2fa7344f4238a9c985224585459d8ab710a23d0dd2176c3743470dda1f3fbd8a83d5e194285f367c7d0c83c3a97c4c4529f94c29cdb6e674283ba75ce2f03@51.77.153.43:30304 because world state is not available at common ancestor at block 15267497 (0x6e7b0900cd580f81b65454eba1e0301d43f70c674f8cee772c60c8d4c0540427)
ethereum_1  | 2022-08-13 08:21:36.577+00:00 | nioEventLoopGroup-3-6 | WARN  | FullSyncTargetManager | Disconnecting PeerId 0x944fc4b766512d031c358cceeb71202679a6eae18f1c0405879d158c0b0f97c8c77cc47cbb2e210e179fcbd94836ed0a513ddeb0f5d95893673dcc15666b163b, reputation PeerReputation 161, validated? true, disconnected? false, client: Geth/v1.10.20-stable-8f2416a8/linux-amd64/go1.18.1, connection 1580445250, enode enode://944fc4b766512d031c358cceeb71202679a6eae18f1c0405879d158c0b0f97c8c77cc47cbb2e210e179fcbd94836ed0a513ddeb0f5d95893673dcc15666b163b@73.119.69.118:30303 because world state is not available at common ancestor at block 15267497 (0x6e7b0900cd580f81b65454eba1e0301d43f70c674f8cee772c60c8d4c0540427)
ethereum_1  | 2022-08-13 08:23:15.446+00:00 | nioEventLoopGroup-3-6 | WARN  | FullSyncTargetManager | Disconnecting PeerId 0x9fc3d924e437b1009f22d95cf906fc966feba3b8cb5d2d1637dab7cf237169b777094047807d966f31d20924165f9498ff6b2446183169dcaf362f3b78b3d81d, reputation PeerReputation 172, validated? true, disconnected? false, client: Geth/v1.10.21-stable-67109427/linux-amd64/go1.18.4, connection 163783682, enode enode://9fc3d924e437b1009f22d95cf906fc966feba3b8cb5d2d1637dab7cf237169b777094047807d966f31d20924165f9498ff6b2446183169dcaf362f3b78b3d81d@63.251.15.85:20010 because world state is not available at common ancestor at block 15267497 (0x6e7b0900cd580f81b65454eba1e0301d43f70c674f8cee772c60c8d4c0540427)
```
